e2e/runner/upgrade.go

```diff
@@ -31,7 +34,7 @@ func (r *E2ERunner) RunGatewayUpgradeTestsExternalChains(conf config.Config, opt
 		r.SolanaVerifyGatewayContractsUpgrade(conf.AdditionalAccounts.UserSolana.SolanaPrivateKey.String())
 	}
 
-	if opts.TestSui {
+	if opts.TestSui && !r.IsRunningUpgrade() {
```

Comment:
Not entirely sure why the gateway upgrade for sui is failing , still looking into it